src/contact.js

```diff
@@ -1,53 +1,47 @@
 import { useState } from "react";
 
 const Contact = () => {
+  const [count, setCount] = useState(0);
+  const [school, setSchool] = useState("First School");
 
-const [count, setCunt] = useState(0)
-const [school, setSchool] = useState("First School");
+  
 
+  const handleHover = () => {
+    console.log("Mouse has entered");
+  };
 
+  const handleLeave = () => {
+    console.log("Mouse has left");
+  };
 
-const value = 20 + 30;
+  const handleClick = () => {
+    setSchool("Second School");
+  };
```

Comment:
Please clear off all these functionalities that were used for instructional purposes as well